shlr/gdb/include/reg/riscv.h

```diff
@@ -0,0 +1,53 @@
+return strdup (
+	"=PC	pc\n"
+	"=SP	sp\n" // ABI: stack pointer
+	"=LR	ra\n" // ABI: return address
+	"=BP	s0\n" // ABI: frame pointer
+	"=A0	a0\n"
+	"=A1	a1\n"
+	"=A2	a2\n"
+	"=A3	a3\n"
+	"=A4	a4\n"
+	"=A5	a5\n"
+	"=A6	a6\n"
+	"=A7	a7\n"
+	"=R0	a0\n"
+	"=R1	a1\n"
+	"=SN	a7\n" // ABI: syscall numer
+	"gpr	zero	.32	0	0\n" // seems to be allocated in the arena too
```

Comment:
Should be .64 according to `Unprivileged Spec v. 20191213` 5.1 
> (RV64I widens the integer registers and supported user address space to 64 bits (XLEN=64 in Figure 2.1).

XLEN is the same for all registers including x0/zero in that figure. Also in gdb:
https://github.com/bminor/binutils-gdb/blob/1fa1262d5a865a0f52c381f6465a3965e95c7904/gdb/testsuite/gdb.arch/riscv-tdesc-loading-02.xml#L6